config/routes.rb

```diff
@@ -2,7 +2,9 @@
   root to: "users#index"
 
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show] do
+    resources :posts, only: [:index, :new, :create, :show] do
+      resources :comments, only: [:create]
```

Comment:
- [ ] Similarly, kindly ensure that there is a pair of corresponding routes in the `config/routes.rb` file for  `new` and `create`, actions in the comments controller. Viewing these routes in the command line should be similar to:
```ruby
   POST   /posts/:post_id/comments(.:format)     comments#create
   GET   /posts/:post_id/comments/new(.:format)     comments#new
```